test/test-applications/integrations/aspnet/Samples.AspNet/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM mcr.microsoft.com/dotnet/framework/aspnet:4.8-windowsservercore-ltsc2019
```

Comment:
@RassK I think you may be running into the issue presented here, and here's a link to a workaround script you can run at startup: https://github.com/docker/for-win/issues/1976#issuecomment-585423014

The issue is present only on windowsservercore docker images. If it was possible to switch to a nanoserver image you could switch to that, but I don't think there are aspnet images based on nanoserver...

Reply:
I am surprised that you are still running into issues after setting up a docker network like you did because the linked issue pertains to the internal DNS resolving `gateway.docker.internal` and `host.docker.internal`, but maybe it is worth trying the workaround

Reply:
In a mean time I simplified it a bit. Creating it's own nat network with fixed gateway address, but even a fix IP could not get connection up. I have created 2 labs W10 based and WS2019 based, both work locally, but in GHA it just don't connect.

Reply:
Update: I can see that host VM definitely attaches the gateway IP and health check request from host to host gateway endpoint works. So the questions remains what blocks the request from container to gateway IP.

Reply:
Update: Seems not a network issue too. Thanks to the Github support verified that nanoserver image can run curl against healt-check endpoint. Still not sure why request gets blocked in Github actions with windows server core image.

Reply:
@pellared @zacharycmontoya from the latest updates seems it's a classic issue: Profiler is not attaching. Somehow in local environments it's fine, but specifically in Github Actions Windows VM + profiled application in docker (aspnet container) causing a problem. Could you have a look?